### PR TITLE
tests/main/snap-seccomp-syscalls: install gperf

### PR DESCRIPTION
**Pull request by bboozzoo**
Update the test to install gperf, since it's no longer preinstalled in our
images.

### CODE REVIEW

File: tests/main/snap-seccomp-syscalls/task.yaml [hunk 1]
```diff
@@ -18,6 +18,9 @@ restore: |
     rm -f syscalls.*
 
 execute: |
+    # XXX: this should be part of our images
+    apt install gperf -y
```
Comment (mvo5) [inline tests/main/snap-seccomp-syscalls/task.yaml]: Should we add a "apt remove" in the restore just for cleanliness?

Comment (bboozzoo) [reply tests/main/snap-seccomp-syscalls/task.yaml]: I think we can leave it behind for now. It's pretty harmless

File: cmd/snap-seccomp/syscalls/syscalls.go [hunk 2]
```diff
@@ -255,17 +255,18 @@ var SeccompSyscalls = []string{
 	"oldolduname",
 	"oldstat",
 	"olduname",
-	"oldwait4",
```
Comment (bboozzoo) [inline cmd/snap-seccomp/syscalls/syscalls.go]: See the commit message for details.

File: tests/main/snap-seccomp-syscalls/task.yaml (hunk 1 above)
Comment (zyga) [reply tests/main/snap-seccomp-syscalls/task.yaml]: Mild preference for being tidy.

File: tests/main/snap-seccomp-syscalls/task.yaml [hunk 3]
```diff
@@ -18,6 +18,9 @@ restore: |
     rm -f syscalls.*
```
Comment (zyga) [inline tests/main/snap-seccomp-syscalls/task.yaml]:
```suggestion
    apt remove -y gperf
```

**Comment by jdstrand** [comment]
LGTM

**Pull request merged**